miner/src/pool/mod.rs

```diff
@@ -108,6 +108,18 @@ impl Priority {
 	}
 }
 
+/// Scoring properties for verified transaction.
+pub trait ScoredTransaction {
```

Comment:
Maybe just extend `txpool::VerifiedTransaction` to simplify the bound in scoring implementation?

Reply:
I didn't want to add methods to the base interface because they are (IMO) related to the miner's functionality only and that's why you used them only in miner's scoring. So I didn't want to "garbage" API of base VerifiedTransaction